Assets/Mirror/Editor/Weaver/Processors/MessageClassProcessor.cs

```diff
@@ -23,10 +36,18 @@ public static void Process(TypeDefinition td)
         static void GenerateSerialization(TypeDefinition td)
         {
             Weaver.DLog(td, "  GenerateSerialization");
+            MethodDefinition existingInterfaceMethod = null;
```

Comment:
Maybe move this section into a method to reduce cognitive complexity

Reply:
To keep existing functionality without repeating code i don't see a good way, do you mean something like this?
MethodDefinition existingMethod = td.Methods.FirstOrDefault(md=>md.Name == "Serialize");
if (existingMethod != null && !existingMethod.Body.IsEmptyDefault())
{
    return;
}
This atleast makes it clear that it works on existing methods, as i think empty MessageBase overrides are handled too (not tested though)

Reply:
Sure,  that is certainly more readable.